crates/handlers/src/graphql/mod.rs

```diff
@@ -106,9 +123,9 @@ pub fn schema(
         homeserver_connection: Arc::new(homeserver_connection),
         site_config,
     };
-    let state: mas_graphql::BoxState = Box::new(state);
+    let state: crate::graphql::state::BoxState = Box::new(state);
```

Comment:
```suggestion
    let state: self::state::BoxState = Box::new(state);
```

or actually import `BoxState` not just in `#[cfg(test)]`?